modules/emr/main.tf

```diff
@@ -102,6 +102,11 @@ EOF
       "Properties": {
         "hive.metastore.client.factory.class": "com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"
       }
+    },
+    {
+      "Classification": "spark-hive-site",
+      "Properties": {
+        "hive.metastore.client.factory.class":"com.amazonaws.glue.catalog.metastore.AWSGlueDataCatalogHiveClientFactory"      }
```

Comment:
What is this for?

Reply:
This is for Spark to be able to use the Glue catalog. I think it will be good to have this config already set since we are already installing spark on these clusters.